rlclientlib/vw_model/safe_vw.cc

```diff
@@ -138,9 +138,11 @@ namespace reinforcement_learning {
       scores[i] = predictions[i].score;
     }
 
-    // push examples back into pool for re-use
-    for (auto&& ex : examples)
+    // push examples back into pool for re-use and clean up examples
+    for (auto&& ex : examples) {
       _example_pool.emplace_back(ex);
+      ex->pred.a_s.delete_v();
```

Comment:
Clean up the example before putting it back in the pool?

Reply:
code and comment are updated.